Mixpanel/Mixpanel.m

```diff
@@ -1367,6 +1367,19 @@ - (void)checkForNotificationsWithCompletion:(void (^)(NSArray *notifications))co
     }];
 }
 
+- (void)checkForNewNotificaitonsWithCompletion:(void (^)(BOOL available))completion
+{ 
```

Comment:
This looks like the same code as `checkForNotificationsWithCompletion:` only just returning a `BOOL`. Couldn't the customer just implement this in their code? Additionally, I'm concerned that the name of the function might be confusing for developers.